app/move/controllers/assign/confirmation.js

```diff
@@ -0,0 +1,29 @@
+const allocationService = require('../../../../common/services/allocation')
+
+const PersonAssignBase = require('./base')
+
+class ConfirmationController extends PersonAssignBase {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setUnassignedMove)
+  }
+
+  async setUnassignedMove(req, res, next) {
+    try {
+      const allocationId = res.locals.move.allocation.id
+
+      const allocation = await allocationService.getById(allocationId)
+      const unassignedMove =
+        allocation.moves.filter(
+          move => !move.person || move.person === null
```

Comment:
Won't `!move.person` already return true for `null` values?

ie, this can become:

```javascript
const unassignedMove = allocation.moves.filter(move => !move.person)[0] || undefined
```

It might also read a little easier as something like this too:

```javascript
const unassignedMoves = allocation.moves.filter(move => !move.person)
const unassignedMoveId = unassignedMoves.length ? unassignedMoves[0].id : undefined
```

Reply:
Yes. Think I got confuzzled because we have to pass `{id: null}` to the api. Personally, I prefer not having to check for the length, but switching it